docs/wiki/01 - STUDY PARAMETERS SETUP/01 - Study Variables/01 - Identifiers.md

```diff
@@ -0,0 +1,71 @@
+# Identifiers
+
+## Overview
+By default LORIS provides 2 different identifiers for each participant or subject: 
+
+- **CandID**, also known as the **DCCID**, is a unique randomized 6-digit numeric ID (e.g. '436792') assigned automatically by LORIS upon participant registration. The CandID is not configurable.
+- **PSCID** (Project Study Center ID) is a unique configurable ID. It can be be setup to be either manually entered when registering a participant or automatically generated and usually contains the site abbreviation followed by sequential or randomized characters (e.g. 'MTL0006'), but its exact format is customizable.
+
+The format and the generation of the **PSCID** must be configured by an admin at the setup stage.
+
+## Configuration
+
+### Front End
+_not yet available_
+
+### SQL
+_not yet available_
+
+### Adding identifiers from the config.xml file
+
+Both the format and the generation of PSCIDs can be configured by the project in the `config.xml` file of the `%LORIS_ROOT%/project/` directory.
+
+PSCIDs can be created for new subjects in one of 3 ways: *sequentially generated*, *manually entered*, or *randomly generated*.
+
+1. ***sequential*** generates PSCIDs sequentially for each new participant registered. **(default)**
+
+ ```xml
+ <PSCID>
+     <generation>sequential</generation> 
+     <structure>
+         <seq type="siteAbbrev"/>
+         <seq type="numeric" minLength="4"/>
```

Comment:
@ridz1208 Can you add 'minvalue' and 'maxvalue' details?

Reply:
If you know the details can you either formulate them and post it here or modify directly my branch??

Otherwise I'll have to go dig up the info in the code and it will take llonger

Reply:
```
     <min>0001</min>
     <max>9999</max> 
```
Minimum and maximum values can be specified for sequentially generated PSCIDs. By default sequence will start at the lowest possible values (i.e.: 0000, AAAA).

Reply:
@PapillonMcGill which types does it apply to ? alpha, numeric, all of them ?

Reply:
all of them.

Reply:
I made changes on lines 32, 56 and 69
 - can you confirm the syntax is correct ?
 - also you mentioned in your comments for sequential generation, shouldn't it apply to random as well ?

Reply:
Yes it does also apply to random

Reply:
changes are made